tensorrt_llm/llmapi/llm_args.py

```diff
@@ -1190,7 +1149,7 @@ def _setup_speculative_config(self):
 
                 self.build_config.max_draft_len = self.speculative_config.max_draft_len
 
-                if self.backend not in ['pytorch', '_autodeploy']:
+                if self.backend != 'pytorch':
```

Comment:
Why are we removing the `_autodeploy` check from here?

Reply:
This is a rebase mistake, I will restore it.